src/yb/yql/cql/cqlserver/cql_service.cc

```diff
@@ -44,6 +48,8 @@ DEFINE_int64(cql_service_max_prepared_statement_size_bytes, 128_MB,
 DEFINE_int32(cql_ybclient_reactor_threads, 24,
              "The number of reactor threads to be used for processing ybclient "
              "requests originating in the cql layer");
+DEFINE_int32(password_hash_cache_size, 16, "Number of password hashes to cache. 0 or "
```

Comment:
Ok with making the default a bit higher like 64.

Reply:
@kmuthukk So I'm assuming you want this cache to be enabled by default, correct?

Reply:
Yes - that's correct. @hectorgcr -- the nice thing about your suggestion of this caching strategy is that we should be able to pull this off without on-disk changes. Therefore, would like the new model to be the default.